content/reference/crosswalk-aws/api-gateway.md

```diff
@@ -0,0 +1,1043 @@
+                return {
+                    statusCode: 200,
+                    body: "Hello, API Gateway!",
+                };
+            },
+        }),
+    }],
+})
+
+// Export the auto-generated API Gateway base URL.
+export const url = api.url;
+```
+
+In many cases, you might want to define your application logic in a different set of files than your
+API Gateway infrastructure. Using modules lets you do this easily. For instance, we might have a subdirectory
+named `app/` that contains a `routes.js` file:
+
+```typescript
+import * as awsx from "@pulumi/awsx";
+
+// Define our routes, independent from the API Gateway itself.
+export async function helloHandler(
+        event: awsx.apigateway.Request): Promise<awsx.apigateway.Response> {
+    return {
+        statusCode: 200,
+        body: "Hello, API Gateway!",
+    };
+}
+```
+
+Then for our API Gateway infrastructure definition we can simply write:
+
+```typescript
+import * as awsx from "@pulumi/awsx";
+import * as routes from "./app/routes";
+
+// Define a new GET endpoint that just returns a 200 and "hello" in the body.
+const api = new awsx.apigateway.API("example", {
+    routes: [{ path: "/", method: "GET", eventHandler: routes.helloHandler }],
+})
+
+// Export the auto-generated API Gateway base URL.
+export const url = api.url;
+```
+
+If you prefer to keep your application and infrastructure code in entirely separate projects, you can define
+Lambdas in the usual way and consume them from the API Gateway resources. For instance, if you already have a
+Lambda Function provisioned in AWS, you can reference it using `aws.lambda.Function.get`:
+
+```typescript
+import * as aws from "@pulumi/aws";
+import * as awsx from "@pulumi/awsx";
+
+// Define a new GET endpoint from an existing Lambda Function.
+const api = new awsx.apigateway.API("example", {
+    routes: [{
+        path: "/",
+        method: "GET",
+        eventHandler: aws.lambda.Function.get("get-handler", "your_lambda_id"),
+    }],
+})
+
+// Export the auto-generated API Gateway base URL.
+export const url = api.url;
+```
+
+For more complete information about creating Lambda Functions, please
+[see the Pulumi Crosswalk for AWS Lambda documentation]({{< relref "lambda.md" >}}). Any of the techniques described may be used
+in combination with the `awsx.apigateway.API` class.
+
+### Defining a Static Route Served by S3
+
+A Static Route serves static content from [S3](https://aws.amazon.com/s3/) at an API endpoint.
+
+To ease population of the S3 bucket to serve from, and connecting it to the API Gateway, you specify the local path --
+either a file or an entire directory -- and Pulumi will upload the contents as S3 objects.
+
+Let's say we have a directory `www` containing a single `index.html` file:
+
+```html
+<h1>Hello, API Gateway + S3!</h1>
+```
+
+The following program will create an API Gateway that serves this content at the `/www` URL:
```

Comment:
Did you mean the `/` URL? The path in the code is /.
```suggestion
The following program will create an API Gateway that serves this content at the `/` URL:
```

Reply:
Good catch, thank you!